src/logion/model/locrequest.model.ts

```diff
@@ -1387,7 +1402,7 @@ export class LocRequestFactory {
         this.ensureUserIdentityPresent(description, isUserRequest)
         const request = new LocRequestAggregateRoot();
         request.id = params.id;
-        request.status = params.draft ? "DRAFT" : "REQUESTED";
+        request.status = params.draft ? "DRAFT" : "REVIEW_PENDING";
```

Comment:
I think that the `draft` parameter is set to true when the LO creates a LOC, which means that the status should be `OPEN` and not `REVIEW_PENDING`.

Reply:
No it's choice left to the user to request and submit in one call.

Reply:
Indeed